samples/tab-adaptive-cards/nodejs/Readme.md

```diff
@@ -0,0 +1,123 @@
+# Tab With AdpativeCard Flow
+
+This App talks about the Teams tab which displays Adaptive card with Node JS
```

Comment:
Let's hyperlink to docs: https://docs.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/build-adaptive-card-tabs

Reply:
added